src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -130,7 +130,7 @@ def __init__(
             num_realizations = config_dict.get(ConfigKeys.NUM_REALIZATIONS)
 
             # RUNPATH_KEY
-            run_path = config_dict.get(ConfigKeys.RUNPATH)
+            run_path = config_dict.get(ConfigKeys.RUNPATH, "simulations/realization%d")
```

Comment:
Suggest to stick to the FMU standard, which would include a dash between `realization` and `%d`.

Reply:
Thanks for the suggestion but when looking at `src/clib/lib/include/ert/enkf/enkf_defaults.hpp` the default run path is as follows:
```
#define DEFAULT_RUNPATH "simulations/realization%d"
```
So I am not sure the FMU standard is used in this case, but maybe I am confused

Reply:
Fair point, your line 133 should probably be consistent with what's in `enkf_defaults.hpp`. I thought I had changed it all in https://github.com/equinor/ert/pull/2603 but apparently not.